SampleCode/Python/sample_identify_cross_page_tables.ipynb

```diff
@@ -4,17 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Identify cross-page tables based on rules\n",
+    "# Identify cross-page tables and merge them into one table based on rules\n",
     "\n",
-    "This sample demonstrates how to use the output of Layout model and some business rules to identify cross-page tables. Once idenfied, it can be further processed to merge these tables and keep the semantics of a table.\n",
+    "This sample demonstrates how to use the output of Layout model and some business rules to identify markdown cross-page tables. Once idenfied, it will be processed to merge these markdown tables and keep the semantics of a table.\n",
     "\n",
-    "Depending on your document format, there can be different rules applied to idenfity a cross-page table. This sample shows how to use the following rules to identify cross-page tables:\n",
+    "Depending on your document format, there can be different rules applied to idenfity a cross-page markdown table. This sample shows how to use the following rules to identify cross-page markdown tables:\n",
     "\n",
-    "- If the 2 tables appear in consecutive pages\n",
-    "- And there's only page header, page footer or page number beteen them\n",
-    "- And the tables have the same number of columns\n",
+    "- Vertical layout\n",
+    "    - If the 2 or more tables appear in consecutive pages\n",
+    "    - And there's only page header, page footer or page number beteen them\n",
+    "    - And the tables have the same number of columns\n",
+    "    - These tables could be considered to one vertical table.\n",
     "\n",
-    "You can customize the rules based on your scenario."
+    "- Horizontal layout\n",
+    "    - If there're 2 or more tables appear in consecutive pages\n",
+    "    - And the table's right side is quite close to the right edge of current page\n",
+    "    - And the next table's left side is quite close to the left edge of next page\n",
+    "    - And the tables have the same number of row count\n",
+    "    - These tables could be considered to one horizontal table.\n",
+    "\n",
+    "You can customize the rules based on your scenario.\n",
+    "\n",
+    "**Tip** : In future versions, we will use html to represent tables in markdown soon since markdown doesn't support merged cell. If you run into merged cell issues, you can consider using html as the solution. A reference code can be found here: *https://github.com/anirudhabodhankar/FormRecognizer-/tree/main/Form-Recognizer-Toolkit* ."
    ]
```

Comment:
Please remove the tip part for now. We will make announcement and update this sample once we release the html format.

Reply:
fixed